WordPress/src/main/java/org/wordpress/android/localcontentmigration/LocalAccessTokenProviderHelper.kt

```diff
@@ -7,5 +7,7 @@ import javax.inject.Inject
 class LocalAccessTokenProviderHelper @Inject constructor(
     private val accountStore: AccountStore,
 ): LocalDataProviderHelper {
-    override fun getData(localSiteId: Int?, localEntityId: Int?) = AccessTokenData(token = accountStore.accessToken)
+    override fun getData(localSiteId: Int?, localEntityId: Int?) = AccessTokenData(
+            token = accountStore.accessToken ?: ""
+    )
```

Comment:
Thanks for the ping @develric :+1: 

It's good to know this can be nullable here. Eventually, this may be bubbled and wrapped in a kind of generic `Result` class to provide better error tracking on the resolver side, so it would even be ok to use `!!` here for now, imo, or even slightly better might be `checkNotNull`. But this is ok as is, as this is something that will be tackled in a later PR.